### DISCUSSION TITLE
Delete the 'update-product' mojo

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#1774] [General]
Tycho extras contains a mojo named 'update-product' that is described as follows:

> Quick&dirty way to update .product file to use latest versions of IUs available from specified metadata repositories.

As there seem not a single test for it, it is described as "Quick&dirty" only supports plugins and it uses outdated API I'd like to delete the mojo if no one knows about a use-case for this (and likes to update it).

**Comment by HannesWell** [comment]
No objection from me.
If one wants to use the latest version of a feature/plugin no version should be used.

I'm not even sure if version are properly handled. At least when launching a Product in the IDE the version handling of PDE is not optimal.
